decent-dao/fractal-interface
# Feature  | Parent Token Allocation

## Description
Adds Parent Token Claiming

- Adds Parent Token UI Component to Create SubDAO workflow
- Updates DaoBuilder and UsulBuilder to build and deploy claim contract
- Add UI to DAO Dashboard to claim tokens


## Notes



## Issue / Notion doc (if applicable)



## Testing
- Create a Parent Token DAO
--Allocate tokens to more than one account (double check that allocations are being spilt correctly)
- Create a SubDAO Proposal and allocate an amount to parent Allocation
- After Proposal Passes, visit new DAO


## Screenshots (if applicable)
![localhost_3000_(Desktop) (42)](https://user-images.githubusercontent.com/38386583/222013583-3c8e5413-b4f9-4924-98e6-6b109f185009.png)

*Amount is different as I have made an update to this screenshot (spacing)
![localhost_3000_ (46)](https://user-images.githubusercontent.com/38386583/222225547-9d03dc07-be31-46f0-b003-eba01b76f7fb.png)
